radicle-cob/src/backend/git/change.rs

```diff
@@ -235,13 +236,22 @@ where
 
     {
         let author = repo.signature()?;
-        let timestamp = author.when().seconds() as Timestamp;
+        let mut timestamp = author.when().seconds();
+
         let mut headers = commit::Headers::new();
         headers.push(
             "gpgsig",
             &String::from_utf8(crypto::ssh::ExtendedSignature::from(signature).to_armored())?,
         );
-        let author = commit::Author::try_from(&author)?;
+        let mut author = commit::Author::try_from(&author)?;
+
+        if let Ok(s) = std::env::var(git::RAD_COMMIT_TIME) {
```

Comment:
I don't wanna hold up this PR but I'm wondering if there is a way to ensure that we're using a debug build for this? I'm not sure how the testing framework works and if it uses the debug target of the binary. Would be good to make sure this doesn't run in released code :)

Reply:
Yeah, we do exactly that when checking `RAD_DEBUG`:

```rust
        #[cfg(debug_assertions)]
        if env::var("RAD_DEBUG").is_ok() {
            // Generate a test keypair that is always the same.
            // This is useful for debugging and testing, since the
            // public key is known in advance.
            return KeyPair::from_seed(Seed::new([0xff; 32]));
        }
        KeyPair::generate()
```
All that's needove the if.ed is that pragma above the `if`.

Reply:
Will do this.  Maybe then we could panic on a badly formatted env var?  Since its in debug mode, we can assume the intent on setting it.

Reply:
Yeah

Reply:
Added the panic B)